<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>new Project</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/icon/MM32_Logo.ico</normaloff>:/icon/icon/MM32_Logo.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="base">
      <attribute name="title">
       <string>Base</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Project</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="proName">
            <property name="frame">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_6" columnstretch="1,1">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Programmer Config</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Type</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="programPart">
              <item>
               <property name="text">
                <string>MM32Link_V1</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Link Mode</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="chipLink">
              <item>
               <property name="text">
                <string>SWD</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>JTAG</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Frequence</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="timeFrequency">
              <item>
               <property name="text">
                <string>20MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>500KHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>200KHz</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Reset Type</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="resetModel">
              <item>
               <property name="text">
                <string>HW Reset</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SYSRESETREQ</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>VECTREST</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Chip Choose</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Series</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Size</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>KByte</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QComboBox" name="partModel"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Part No.</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Core</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QComboBox" name="seriesName"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="core">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="memorySize">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>File</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="filePath"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="selectFile">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Descript</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QTextEdit" name="textEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="option">
      <attribute name="title">
       <string>Option</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>391</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="5,5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QCheckBox" name="standby">
          <property name="text">
           <string>Generate a reset event when entering the Standby mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="stop">
          <property name="text">
           <string>Generate a reset event when entering the Stop mode</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Data</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Data0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="data0"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Data1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="data1"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>191</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Watch Dog</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="hdDog">
          <property name="text">
           <string>Hardware watchdog</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sfDog">
          <property name="text">
           <string>Software watchdog</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>210</number>
     </property>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
